websites/J/JAVX/presence.ts

```diff
@@ -0,0 +1,143 @@
+const presence = new Presence({
+    clientId: "830476272978362408",
+    injectOnComplete: true,
+    appMode: true
+  }),
+  strings = presence.getStrings({
+    playing: "general.playing",
+    paused: "general.paused"
+  });
+
+let video = { playback: false, currentTime: 0, duration: 0, paused: false };
+
+presence.on(
+  "iFrameData",
+  (data: {
+    playback: boolean;
+    currentTime: number;
+    duration: number;
+    paused: boolean;
+  }) => {
+    video = data;
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "javx",
+      startTimestamp: Math.floor(Date.now() / 1000)
+    },
+    { pathname, search } = document.location;
+
+  if (search.startsWith("?s")) {
+    const s = new URLSearchParams(search).get("s");
+
+    if (s) {
+      presenceData.details = "Đang tìm phim với từ khoá:";
+      presenceData.state = encodeURI(s);
+    } else {
+      presenceData.details = "Đang tìm phim";
+    }
```

Comment:
```suggestion
    presenceData.details = s ? "Đang tìm phim với từ khoá:" : "Đang tìm phim";
    if(s) presenceData.state = encodeURI(s);
```
Why I did ``if`` because it isn't good to set something in object to null or undefined :P